framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -441,9 +446,9 @@ public final synchronized void init(ProcessingEnvironment env) {
         // This is used to trigger AggregateChecker's setProcessingEnvironment.
         setProcessingEnvironment(env);
 
-        int jreVersion = PluginUtil.getJreVersion();
-        if (jreVersion != 8) {
-            userErrorAbort(
+        double jreVersion = PluginUtil.getJreVersion();
```

Comment:
I don't think this change is correct.
https://github.com/eisop/checker-framework/blob/master/javacutil/src/main/java/org/checkerframework/javacutil/PluginUtil.java#L528
returns an int, so it can never be 1.8.